product_docs/docs/biganimal/release/using_cluster/managing_replicas.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Managing replicas"
+---
+
+### Create a faraway replica
+
+You can create faraway replicas in different regions in your cloud. There is no limit on the number of faraway replicas each cluster can have. However, the added resources and data transfers can increase your cloud costs.
+
+1. Sign in to the [BigAnimal](https://portal.biganimal.com/) portal.
+
+1. Go to the [Clusters](https://portal.biganimal.com/clusters) page. A list of previously created clusters appears.
+
+1. Select the *Faraway Replica* icon next to a cluster to create its faraway replica. The Create Replica page appears.
+
+1. You cannot edit the **Cloud Provider** field. The cloud provider is inherited from the parent cluster. 
+
+1. In the Replica Settings tab, Enter a name for your replica in the **Faraway Replica Name** field.
+
+1. Enter a password for your replica in the **Password** field.
+
+1. You cannot edit the **Database Type** field. The Postgres type and Postgres version are inherited from the parent cluster.  
+
+1. In the **Region** section, select the region where you want to deploy your replica.
+
+ You can select a region for deploying your replica even if it is not yet *Active*. Your replica creation request is added to a queue, and the replica is created after you activate the region. See [Activating regions](/biganimal/latest/getting_started/activating_regions) for more information.
+ 
+1.  The fields in the **Instance Type** section are prepopulated with values inherited from the parent cluster and are editable. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on instance type settings.
+
+1.  The fields in the **Storage** section are prepopulated with values inherited from the parent cluster. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the storage settings.
+ 
+1.  In the **Networking** section, specify whether to use private or public networking. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the Networking settings.
+
```

Comment:
the backup retention period will also be a configurable setting

Reply:
OK. Adding that step here and adding a procedure to Restore replica back up.